### DISCUSSION TITLE
An error if a cracked players joined, premium not showed that error.

### DISCUSSION THREAD

**Discussion by Scoooolzs** [games647/FastLogin#985] [Q&A]
Cracked Players:
![image](https://user-images.githubusercontent.com/84376629/216777910-9ed0b015-d8a4-4c17-85d2-7622918a0c2a.png)

Premium Players:
![image](https://user-images.githubusercontent.com/84376629/216777983-3fbe97e0-4e24-421c-94d5-2148f904e39c.png)

Yeah, you can just ignore that error, but i just want a clear & good console 👍

**Comment by TuxCoding** [comment]
Bug reports should go to the bug template. Nevertheless, we fixed it here: https://github.com/games647/FastLogin/commit/23d7594b6708ebb32ccef5c853ea9915546e21af
